packages/nimble-components/src/table/index.ts

```diff
@@ -537,15 +619,33 @@ export class Table<
         this.$emit('selection-change', detail);
     }
 
+    private selectionStateChanged(): void {
+        this.setSelectionCheckboxState();
+    }
+
+    private selectionCheckboxChanged(): void {
+        this.setSelectionCheckboxState();
+    }
```

Comment:
Are the two above methods called anywhere?

Reply:
By change handlers for FAST observables.